src/lib.rs

```diff
@@ -131,51 +119,39 @@ fn print_message(buf: &[u8]) {
     println!();
 }
 
-fn spawn_cmd_sender<'f>(
-    local_ip: IpAddr,
-    target_ip: IpAddr,
-    port: u16,
-) -> AnyResult<Sender<Vec<u8>>> {
-    let socket = UdpSocket::bind((local_ip, port)).map_err(|e| {
-        anyhow!(
-            "Couldn't bind to local socket {}:{} - {}",
-            local_ip,
-            port,
-            e
-        )
-    })?;
+fn spawn_cmd_sender(local_ip: IpAddr, target_addr: SocketAddr) -> AnyResult<Sender<Vec<u8>>> {
```

Comment:
I don't quite understand this. The `local_ip` is the IP we are listening. Is there a difference between `192.168.1.1` and `127.0.0.1`? Also the `Listen` port is the same as the target port?
We are listening on the same port that we use to send frames to the drone!?

Reply:
in order to send UDP frames over the wire, i need to get a local socket and start shoving packets through it, the target_addr.port() is a bit arbitrary (i need to check it again to be sure though)

The local ip address however has an impact  (again, as i understand it) because I want to listen to frames that are being sent from the drone, which is the reason why I need to figure out which IP belongs to the interface that is connected to the drone.

Does that make sense ? ^^